src/IceNucleation.jl

```diff
@@ -54,4 +67,72 @@ function dust_activated_number_fraction(
     end
 end
 
```

Comment:
In the documentation you mention temperature ranges for which the parameterization is valid. Could we add a similar warning and return zero rate for temperatures outside of that range?

Kind of similar to what we are doing here: https://github.com/CliMA/CloudMicrophysics.jl/blob/3a1b0ab630ffdefb9a83245af29931ff8ae776a6/src/IceNucleation.jl#L46-L54